frontend/src/features/admin-form/create/featureTour/FeatureTour.tsx

```diff
@@ -0,0 +1,53 @@
+import Joyride from 'react-joyride'
+
+import { FEATURE_TOUR } from './constants'
+import { FeatureTourTooltip } from './FeatureTourTooltip'
+
+interface FeatureTourProps {
+  shouldRun: boolean
+}
+
+const featureTourSteps = [
+  {
+    target: `#${FEATURE_TOUR.FIRST_STEP.id}`,
+    title: `${FEATURE_TOUR.FIRST_STEP.title}`,
+    content: `${FEATURE_TOUR.FIRST_STEP.content}`,
+    disableBeacon: true,
+  },
+  {
+    target: `#${FEATURE_TOUR.SECOND_STEP.id}`,
+    title: `${FEATURE_TOUR.SECOND_STEP.title}`,
+    content: `${FEATURE_TOUR.SECOND_STEP.content}`,
+    disableBeacon: true,
+  },
+  {
+    target: `#${FEATURE_TOUR.THIRD_STEP.id}`,
+    title: `${FEATURE_TOUR.THIRD_STEP.title}`,
+    content: `${FEATURE_TOUR.THIRD_STEP.content}`,
+    disableBeacon: true,
+  },
+]
```

Comment:
a little weird for this constant to be created from another constant. can we not use this constant as the exported constant instead?